components/infobox/wikis/overwatch/infobox_map_custom.lua

```diff
@@ -0,0 +1,74 @@
+---
```

Comment:
The country/location doesn't show a Flag, like the current line version does

Reply:
Most wikis that have location on Infobox Map currently have no flags and just show the plain wiki text. Along with that the few that do show flags, some of them have the flag added directly on the page and not on the template or module itself. I left both off since most maps aren't based on a real place, and it would be unrealistic to add fictional game flags to the maps

Reply:
Yeah, but you can add an override for Overwatch, if the Overwatch locations always are real countries.

Reply:
The biggest question I have with that is: ``Does having the flag and or the link to the nations category, on the location field provide any actual benefit if it only just links to the country category?`` I didn't see a huge benefit from having it thus why I left it off entirely. I have asked the contributors on the wiki about their [preference](https://discord.com/channels/93055209017729024/927989478071611402/943239630696882196), will adjust the infobox according to their answer